shipyard_blueprints/airtable/shipyard_airtable/cli/upload.py

```diff
@@ -0,0 +1,118 @@
+import sys
+import pandas
```

Comment:
This needs to include one additional argument for `Insert Method`, which is a choice of `replace` or `append`. Right now I'm assuming that it just overwrites the existing data?

Reply:
For clarification this is currently doing an upsert. 

Do we still want to add the replace or append. The end user will able to accomplish replace or append by omitting or including the record id unless replace is intended to clear out the table entirely so that the only data that exists what is in the csv file upload

Reply:
Yes - to match functionality of our other uploads we should add in the ability to overwrite an existing table with the data being uploaded. [](edited-with-Axolo)

Reply:
I would argue that upsert is a useful and intuitive function, should I include it as an option considering not all services may not have an endpoint that supports that out of the box?

Reply:
Yeah thats fine [](edited-with-Axolo)